src/jit/codegenarm.cpp

```diff
@@ -1098,13 +1098,13 @@ void CodeGen::genLockedInstructions(GenTreeOp* treeNode)
 //             which invoke push {tmpReg} N times.
 //          2) Fore /o build  However, we tickle the pages to ensure that SP is always
 //             valid and is in sync with the "stack guard page". Amount of iteration
-//             is N/PAGE_SIZE.
+//             is N/GetOsPageSize().
 //
 // Comments:
 //      There can be some optimization:
 //          1) It's not needed to generate loop for zero size allocation
 //          2) For small allocation (less than 4 store) we unroll loop
-//          3) For allocation less than PAGE_SIZE and when it's not needed to initialize
+//          3) For allocation less than GetOsPageSize() and when it's not needed to initialize
```

Comment:
This should not be OS page size for native images. The native images should not depend on the actual OS page size.

These comments should be updated to refer to eeGetPageSize().